createdisk.sh

```diff
@@ -326,6 +326,10 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo podman pull quay.io/crcont
 # Stop the kubelet service so it will not reprovision the pods
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl stop kubelet
 
+# Remove system reserved block from kubelet config
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_${yq_ARCH} -o /usr/local/bin/yq && chmod +x /usr/local/bin/yq
```

Comment:
Delete yq after that? or perhaps do it on the host?

Reply:
This can be done on the host with:
```
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} cat /etc/kubernetes/kubelet.conf | ${YQ} delete - systemReserved |  ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} sudo tee /etc/kubernetes/kubelet.conf
```
which imo is nicer than downloading the binary on the VM.

Reply:
Nice! I updated the PR.